docs/modules/ROOT/pages/release-notes/release-notes-2021-7.adoc

```diff
@@ -0,0 +1,29 @@
+= Release Notes - Payara Platform Community 5.2021.7
+
+== Supported APIs and Applications
+
+* Jakarta EE 8
+* Java EE 8 Applications
+* Jakarta EE 9
+* MicroProfile 4.1
+
+=== New Features
+* [https://github.com/payara/Payara/pull/5398[FISH-5646]] Add Client Certificate Validation API
+
+=== Bug Fixes
+* [https://github.com/payara/Payara/pull/5399[FISH-5711]] Fix Could Not Load Formatter Class Error in Payara Micro
+* [https://github.com/payara/Payara/pull/5397[FISH-5701]] Application does not deploy anymore on Payara Server Docker image
+* [https://github.com/payara/Payara/pull/5374[FISH-5695]] ArrayIndexOutOfBoundsException when the ConfigProperty value ends in a dollar sign \[Community Contribution\]
+* [https://github.com/payara/Payara/pull/5395[FISH-5693]] Delimiter for configuring multiple KeyStores and TrustStores must use JVM Platform Delimiter
+* [https://github.com/payara/Payara-Enterprise/pull/447[FISH-5691]] Do not propagate non-transactional EM to managed tasks
+* [https://github.com/payara/Payara-Enterprise/pull/447[FISH-5690]] Remove duplicate postInvoke handler invocation on failure
```

Comment:
Remove Duplicate postInvoke Handler Invocation on Failure

-Note: Should it be PostInvoke ? not sure if this is one that has to be written a certain way/name